qa/L0_output_name/output_name_test.py

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import argparse
+import numpy as np
+import os
+from builtins import range
+from functools import partial
+from PIL import Image
+
+import grpc
+from tensorrtserver.api import api_pb2
+from tensorrtserver.api import grpc_service_pb2
+from tensorrtserver.api import grpc_service_pb2_grpc
+import tensorrtserver.api.model_config_pb2 as model_config
+
+import json
+import requests
+import unittest
+FLAGS = None
+
+class OutputNameValidationTest(unittest.TestCase):
+    def TestGRPC(self):
+        channel = grpc.insecure_channel(self.url)
+        grpc_stub = grpc_service_pb2_grpc.GRPCServiceStub(channel)
+        # Prepare request for Status gRPC
+        request = grpc_service_pb2.StatusRequest(model_name=self.model_name)
+        response = grpc_stub.Status(request)
+
+        request_ = self.requestGenerator("DUMMY", FLAGS)
+        # Send request
+        response_ = grpc_stub.Infer(request_)
+        return str(response_).find("code: SUCCESS")==-1
+
+    def TestHTTP(self):
```

Comment:
I don't see how you can test HTTP using grpc? Just using grpc (to avoid using our client library) is suffuicient, so you should need this test at all.

Reply:
Should I remove the HTTP test entirely then?

Reply:
yes